pennylane/interfaces/tf.py

```diff
@@ -98,10 +101,20 @@ def __repr__(self):
         metric_tensor = qnode.metric_tensor
         draw = qnode.draw
         func = qnode.func
+        num_variables = property(lambda self: qnode.num_variables)
+        arg_vars = property(lambda self: qnode.arg_vars)
 
     @qnode_str
     @tf.custom_gradient
     def _TFQNode(*input_, **input_kwargs):
+        # determine which input tensors are being recorded for backpropagation
+        requires_grad = [
+            tape_lib.should_record_backprop([1 * i])
```

Comment:
Why is the `1` needed? :thinking:

Reply:
It seems that this function only accepts tensors, not Variables. Generally, this shouldn't be an issue; `Variable` subclasses from `tf.core.Tensor`, so via duck typing all TF functions/classes that accept tensors should also work with Variables. But maybe this function is too low-level/close to C++ for duck typing to work?

Regardless, multiplying a Variable by a literal is a quick way of converting it into a tensor without breaking the computational graph, so that it will be accepted by this method.

I suppose it's clearer/more explicit to simply do `tf.convert_to_tensor(i)`.
```suggestion
            tape_lib.should_record_backprop([tf.convert_to_tensor(i)])
```